ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -30,15 +30,31 @@
 
 #include <fstream>
 
+namespace gcs = google::cloud::storage;
```

Comment:
we can do using namespace gcs = google:\:cloud\::storage directly here, instead of adding using inside the function.

Reply:
addressed here: [600b09f](https://github.com/wiredtiger/wiredtiger/pull/8717/commits/600b09f6e6320c6b3a271bf48c7bbe49933a1788)

Reply:
we can directly combine the two lines as well.

Reply:
combining the two lines will give syntax error, so I would leave them seperate.